.gitignore

```
@@ -40,7 +40,8 @@ typings/
 *.tgz
 
 # dotenv environment variables file
-.env
+.env	.env
+.env.*
```

```suggestion
.env*
```
Shouldn't that be enough?